apis/apis/src/main/java/com/expensetracker/apis/CustomException/ExpenseException.java

```diff
@@ -0,0 +1,9 @@
+package com.expensetracker.apis.CustomException;
+
+public class ExpenseException extends Exception {
+
```

Comment:
we can also have errorCode here, to define Exception more clearly

Reply:
Added